src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -198,7 +198,7 @@ internal bool ValidKeySize()
         /// <inheritdoc/>
         public override bool Sign(ReadOnlySpan<byte> input, Span<byte> signature, out int bytesWritten)
         {
-            if (input == null || input.Length == 0)
```

Comment:
Comparing a span to 'null' might be redundant, 'null' literal will be implicitly converted to a 'Span<T>.Empty' (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2265

Reply:
interesting. thanks.